stock_orderpoint_manual_procurement/wizards/make_procurement_orderpoint.py

```diff
@@ -68,7 +68,7 @@ def make_procurement(self):
             values = item.orderpoint_id._prepare_procurement_values(item.qty)
             # Run procurement
             try:
-                self.env['procurement.group'].run(
+                self.env['procurement.group'].with_context(by_pass=True).run(
```

Comment:
@bodedra It is not recommended to pass parameters through context. You could face performance problems as whole environment will be recreated.

You can add parameter 'by_pass' to run function.

Reply:
@rousseldenis Corresponded. Thank you.